examples/gnn_cost_model/utils/db_utils.py

```diff
@@ -0,0 +1,26 @@
+import sqlite3
```

Comment:
Consider merging into `examples/gnn_cost_model/compiler_gym_dataset.py`, I don't think these functions are used anywehere else (ignore me if I'm wrong!)

Reply:
Yes they aren't, they were initially used elsewhere before when I started this, but since changed it. Thanks!